signup.js

```diff
@@ -0,0 +1,131 @@
+let signupValid = false
+
+//ID 선택하기
+const emailInputElement = document.getElementById('email');
+const nickInputElement = document.getElementById('nickname');
+const pwInputElement = document.getElementById('pw');
+const pwCheckInputElement = document.getElementById('pwcheck');
+const errorMessage = document.querySelector('.error-message');
```

Comment:
P2:
해당 변수도 errorMessage가 들어갈 요소이니 아래와 같은 이름이 더 유의미할 것 같습니다.

```suggestion
const errorMessageElement = document.querySelector('.error-message');
```